build.sh

```diff
@@ -131,8 +131,12 @@ chmod 755 $DESTINATION/sites/default
 
 # Inculde copies of the settings files that were used to build the site, for reference
 SETTINGS_SITE="$DESTINATION/profiles/$PROJECT/settings"
+
+# replace PROJECT with actual project name in local.settings.php
+sed -i '' "s/PROJECT/$PROJECT/g" $DESTINATION/profiles/$PROJECT/scripts/settings/local.settings.php
```

Comment:
I'm not exactly sure what those lines of code are trying to do, but I'm going to say that we probably don't need them.